using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-api-reference.md

```diff
@@ -1,150 +1,271 @@
+### extensionVersion
 
-To get the version of the Campaign Standard extension, use the following code sample:
+Returns the running version of the AEPCampaign extension.
 
 {% tabs %}
 {% tab title="Android" %}
-### Java
 
-### Syntax
+**Syntax**
 
 ```java
 public String extensionVersion()
 ```
 
-### Example
+**Example**
 
 ```java
 Campaign.extensionVersion();
 ```
+
+{% endtab %}
+{% tab title="iOS (AEP 3.x)" %}
+
+**Syntax**
+
+```swift
+static var extensionVersion: String
+```
+
+**Example**
+
+**Objective-C**
+
+```objc
+NSString *campaignVersion = [AEPMobileCampaign extensionVersion];
+```
+
+**Swift**
+
+```swift
+let campaignVersion = Campaign.extensionVersion
+```
+
 {% endtab %}
 
-{% tab title="iOS" %}
-### Syntax
+{% tab title="iOS (ACP 1.x)" %}
+
+**Syntax**
 
-```objectivec
+```objc
 + (nonnull NSString*) extensionVersion;
 ```
 
-### Examples
+**Example**
 
-### Objective C
+**Objective-C**
 
-```objectivec
-NSLog(@"ACPCampaign version: %@", [ACPCampaign extensionVersion]);
+```objc
+NSString *campaignVersion = [ACPCampaign extensionVersion];
 ```
 
-### Swift
+**Swift**
 
 ```swift
-print("ACPCampaign version: ", ACPCampaign.extensionVersion())
+let campaignVersion = ACPCampaign.extensionVersion()
 ```
+
 {% endtab %}
 
 {% tab title="React Native" %}
-### JavaScript
 
-### Syntax
+**Syntax**
 
 ```javascript
 extensionVersion(): Promise<string>
 ```
 
-### Example
+**Example**
 
 ```javascript
 ACPCampaign.extensionVersion().then(version => console.log("AdobeExperienceSDK: ACPCampaign version: " + version));
 ```
+
 {% endtab %}
+
 {% endtabs %}
 
-## resetLinkageFields
+---
+
+### registerExtension
 
-This method clears the cached rules from the previous download before triggering a rule download request to the configured Campaign server. If the current SDK privacy status is not OPT\_IN, no rules download occurs.
+Registers the Campaign extension with the Mobile Core.
 
 {% tabs %}
 {% tab title="Android" %}
-### Java
 
-### Syntax
+**Syntax**
+
+```java
+public static void registerExtension()
+```
+
+**Example**
+
+```java
+Campaign.registerExtension();
+```
+
+{% endtab %}
+{% tab title="iOS (AEP 3.x)" %}
+
+This API no longer exists in `AEPCampaign`. Instead, the extension should be registered by calling the `registerExtensions` API in the MobileCore. Please see the updated SDK initialization steps at the [migrate to Swift tutorial](https://aep-sdks.gitbook.io/docs/resources/migrate-to-swift).
+
+{% endtab %}
+
+{% tab title="iOS (ACP 1.x)" %}
+
+**Syntax**
+
+```objc
++ (void) registerExtension;
+```
+
+**Example**
+
+**Objective-C**
+
+```objc
+[ACPCampaign registerExtension];
+```
+
+**Swift**
+
+```swift
+ACPCampaign.registerExtension()
+```
+
+{% endtab %}
+
+{% tab title="React Native" %}
+
+**Syntax**
+
+```javascript
+registerExtension()
+```
+
+**Example**
+
+```javascript
+ACPCampaign.registerExtension();
+```
```

Comment:
For RN we should indicate the extension should be registered in the native implementation:
```
When using React Native, register the Campaign extension with Mobile Core in native code as shown on the Android and iOS tabs.
```